pkg/types/example_codec_test.go

```diff
@@ -0,0 +1,47 @@
+package types_test
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/codec/encodings"
+	"github.com/smartcontractkit/chainlink-common/pkg/codec/encodings/binary"
+)
+
+// ExampleCodec provides a minimal example of constructing and using a codec. For more detail, refer to the Example in
+// pkg/codec/example_test.go.
```

Comment:
If we're going to commit to beefing up the godoc (yes please!), I don't think you need the extra reference here. Once we start answering questions by linking to the docs people will easily discover the other examples from the rendered docs.
```suggestion
// ExampleCodec provides a minimal example of constructing and using a codec.
```